contracts/src/finance/vesting_wallet.rs

```diff
@@ -107,12 +107,16 @@ pub struct VestingWallet {
     /// [`Ownable`] contract.
     pub ownable: Ownable,
     /// Amount of Ether already released.
+    #[allow(clippy::used_underscore_binding)]
```

Comment:
I can see some inconsistency in attributes' naming, e.g. `ownable` & `_released`. We should fix it with #386.